src/yaml/mapping.cr

```diff
@@ -29,16 +29,15 @@ module YAML
   #
   # ```
   # employee = Employee.from_yaml("title: Manager\nname: John\nage: 30")
-  # employee.age # => undefined method 'age'.
+  # employee.age # undefined method 'age'. (compile error)
   #
-  # employee = Employee.from_yaml("title: Manager")
-  # # => ParseException: missing yaml attribute: name
+  # Employee.from_yaml("title: Manager") # raises YAML::ParseException
   # ```
   #
   # You can also define attributes for each property.
   #
   # ```
-  # class Employee
+  # class Employer
```

Comment:
@maiha I think using `Employer` is fine. The benefits of being able to check that example code compiles and runs well outweigh the difference in the name.

We could also use a totally different name here, like Elephant, because it's just showing another way of using `YAML.mapping` :-)